snapcraft/tests/plugins/test_dump.py

```diff
@@ -170,7 +170,7 @@ def test_dump_symlinks_to_libc(self):
 
         # Even though this symlink is absolute, since it's to libc the copy
         # plugin shouldn't try to follow it or modify it.
-        libc_libs = snapcraft.repo.get_pkg_libs('libc6')
+        libc_libs = snapcraft.repo.Repo.get_package_libraries('libc6')
```

Comment:
`libc6` goes by different names in different distros. For example, RPM based distributions call this `glibc`.